packages/rocketchat-authorization/server/startup.js

```diff
@@ -73,6 +74,7 @@ Meteor.startup(function() {
 	const defaultRoles = [
 		{ name: 'admin',     scope: 'Users',         description: 'Admin' },
 		{ name: 'moderator', scope: 'Subscriptions', description: 'Moderator' },
+		{ name: 'leader', 	 scope: 'Subscriptions', description: 'Leader' },
```

Comment:
Can you fix the spaces here to keep the indentation? Seems that you are using tabs